src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -443,6 +475,8 @@ const AppWrapper = (props: { children: ReactNode }) => {
     Agent.agent.keriaNotifications.onRemoveNotification((event) => {
       notificatiStateChanged(event, dispatch);
     });
+    Agent.agent.connections.onConnectionRemoved();
+    Agent.agent.connections.onResolveOobi();
```

Comment:
Please move these 2 lines to the start of the `async start` function in `agent.ts`.

These are not meant to be setup by the UI - these are core related event listeners.